starlight_help/src/content/docs/change-your-password.mdx

```diff
@@ -20,6 +20,6 @@ account.
 <FlattenedSteps>
```

Comment:
This isn't new, but I feel like "You can reset your password ..."  in the copy at the top of the file is not using standard terminology. Typically, "reset" refers to when you are resetting via email, and "change your password" when you do it in the UI.

Reply:
Is there anything in particular you recommend changing on this page? I think it's ok to use "reset" or "change" when talking about both flows together.

Reply:
Oh, I think this comment is just mipslaced. Fixing elsewhere.